src/libraries/System.Private.CoreLib/src/System/Double.cs

```diff
@@ -1671,47 +1671,310 @@ static double NegativeN(double x, int n)
         /// <inheritdoc cref="ITrigonometricFunctions{TSelf}.Acos(TSelf)" />
         public static double Acos(double x) => Math.Acos(x);
 
+        /// <inheritdoc cref="ITrigonometricFunctions{TSelf}.AcosPi(TSelf)" />
+        public static double AcosPi(double x)
+        {
+            return Acos(x) / Pi;
```

Comment:
This will emit a `vdivsd`?
Perf-wise would it be better to have a constant `1d / Pi` so that a `vmulsd` can be used.

Reply:
Perf isn't all the matters, correctness and accuracy matter as well.

This is a reasonable implementation for .NET 7 that will likely be replaced with a more accurate implementation for .NET 8